internal/hcs/system.go

```diff
@@ -339,6 +339,20 @@ func (computeSystem *System) Wait() (err error) {
 	return nil
 }
 
+// WaitExpectedError synchronously waits for the compute system to shutdown or terminate, and ignores the passed error if it occurs.
```

Comment:
NIT: Install ReWrap for VSCode and it will help to leave comments in an easier to read length (80 chars)